tests/EdgeToEdge/Routes/DisplayPageRouteTest.php

```diff
@@ -19,7 +20,17 @@
 class DisplayPageRouteTest extends WebRouteTestCase {
 
 	public function testWhenPageHasCustomTemplate_customTemplateIsRendered(): void {
-		$client = $this->createClient( [ 'skin' => [ 'default' => 'cat17' ] ] );
+		$client = $this->createClient(
+			[],
+			function ( FunFunFactory $factory ): void {
```

Comment:
Would you object to making the code more DRY by creating a private method in the test like `setDefaultSkin( FunFunFactory $factory)`?